src/Domain.LinnApps/LinnWeek.cs

```diff
@@ -2,12 +2,16 @@
 {
     using System;
 
+    using Linn.Purchasing.Domain.LinnApps.Forecasting;
+
     public class LinnWeek
     {
         public int WeekNumber { get; set; }
 
         public DateTime StartsOn { get; set; }
 
         public DateTime EndsOn { get; set; }
+
+        public ForecastWeekChange ForecastChange { get; set; }
```

Comment:
this can't be a one to one relation can it?

Reply:
I had assumed it was! One % change per week I thought? Or can a week have multiple different values?

Reply:
oh yeah. you're completely correct of course.